app/components/password_confirmation_component.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= content_tag(:'lg-password-confirmation', **tag_options) do %>
+  <%= render ValidatedFieldComponent.new(
+        form: form,
+        name: :password,
+        type: :password,
+        label: default_label,
+        required: true,
+        **field_options,
+        input_html: field_options[:input_html].to_h.merge(
+          id: input_id,
+          class: ['password-confirmation__input', *field_options.dig(:input_html, :class)],
+        ),
+      ) %>
+  <%= render ValidatedFieldComponent.new(
+        form: form,
+        name: :password_confirmation,
+        type: :password_confirmation,
+        label: confirmation_label,
+        required: true,
+        **field_options,
+        input_html: field_options[:input_html].to_h.merge(
+          id: input_confirmation_id,
+          class: ['password-confirmation__input-confirmation', *field_options.dig(:input_html, :class)],
```

Comment:
There appears to be a large gap between the password confirmation field and the toggle, vs. what's in the design.

Implementation|Design
---|---
![image](https://user-images.githubusercontent.com/1779930/234277948-914104a7-6582-4ec4-9080-f8b89eb28e92.png)|![image](https://user-images.githubusercontent.com/1779930/234278051-3f794444-499f-46c0-869d-15514c126860.png)

I think it's coming from the default bottom margin on SimpleForm fields, which we could override.

```suggestion
          class: ['password-confirmation__input-confirmation', 'margin-bottom-0', *field_options.dig(:input_html, :class)],
```